src/test/java/org/broadinstitute/hellbender/IntelDeflaterIntegrationTest.java

```diff
@@ -94,64 +94,4 @@ public void deflateInflateWithIntelDeflater() {
             Assert.assertEquals(input, result);
         }
     }
-
-    @Test(enabled = false)
-    public void bamWriteReadWithIntelDeflater() throws IOException {
```

Comment:
This test was deleted here because we already run this test as an integration test in gkl. I believe it was disabled here because of issues running on architectures that do not support `IntelInflater`. We can add a bam based test to gatk.